.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,16 +19,17 @@ _You may update this checklist before and/or after creating the PR. If you're un
 - [ ] Changes are limited to a single goal (no scope creep)
 - [ ] The feature branch you're submitting as a PR is up to date (merged) with the latest `dev` branch
 - [ ] Changes adhere to [PEP-8 Style Guidelines](https://peps.python.org/pep-0008/)
```

Comment:
With the auto-formatters and linters and pre-commit hooks installed by this PR, is this line necessary anymore?

Reply:
Good catch. No need for that line anymore, I'll remove it in my next commit. Thanks.